applicants.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+    <link rel="stylesheet" href="./src/applicants.css">
+</head>
+    <body>
+        <main>
+            <h1>Applicants</h1>
+            <table>
+                <thead>
+                    <tr>
```

Comment:
there is only one row in this ```thead``` tag, however, it seems there are multiple properties on your cat objects. was there a reason for this?